### PR TITLE
Sign-in & Sign up fixed, styling & html added

### PR DESCRIPTION
**Pull request by Rachella**
Mainly html and css added. Fixed sign-in and sign up issues - now functioning.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -26,15 +26,15 @@ var plugins = [
...
-  //   register: require('yar'),
-  //   options: {
-  //     cookieOptions: {
-  //       password: 'passwords',
-  //       isSecure: false
-  //     }
-  //   } 
-  // },
+  { 
+    register: require('yar'),
```
Comment (f3r) [inline index.js:30]: oh i see :)

File: templates/home.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<html>
+<head>
+  <title>PicForEating</title>
+
+  <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+  <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
```
Comment (f3r) [inline templates/home.html:6]: why do you need [jquery-migrate](https://github.com/jquery/jquery-migrate/)?

File: index.js (hunk 1 above)
Comment (Rachella) [inline index.js:30]: The only way I could figure out how to make my page work was to comment this out - something about it was breaking my code. I kept getting bad cookie error.. I finally changed something somewhere else that I could include this back in!

File: templates/home.html (hunk 2 above)
Comment (Rachella) [inline templates/home.html:6]: Not 100% sure what it is used for - thought it was related to API to be used. Was in my notes somewhere...

**Pull request merged**